finality-aleph/src/network/clique/protocols/mod.rs

```diff
@@ -69,6 +72,7 @@ impl<PK: PublicKey> Display for ProtocolError<PK> {
             CardiacArrest => write!(f, "heartbeat stopped"),
             NoParentConnection => write!(f, "cannot send result to service"),
             NoUserConnection => write!(f, "cannot send data to user"),
+            NotAuthorized => write!(f, "user not authorized"),
```

Comment:
Probably "peer" rather than "user", we use the latter to refer to the local user of the network.

Reply:
changed